PURL-TYPES.rst

```diff
@@ -471,6 +471,18 @@ including container images built by Docker and others:
       pkg:oci/static@sha256%3A244fd47e07d10?repository_url=gcr.io/distroless/static&tag=latest
       pkg:oci/hello-wasm@sha256%3A244fd47e07d10?tag=v1
 
+pear
+----
+``pear`` for PEAR and PECL PHP packages:
+
+- The default repository for PEAR is ``https://pear.php.net``. The default repository fo PECL is ``https://pecl.php.net/``.
+- The ``namespace`` is the channel of the package.
+- The ``name`` is not case sensitive
+- Examples:
```

Comment:
your examples suggest, that a version is not required.
if so, please add a note here.